tests/openassetio-core-c/private/managerApi/CManagerInterfaceAdapterTest.cpp

```diff
@@ -70,7 +70,7 @@ SCENARIO("A host calls CManagerInterfaceAdapter::identifier") {
           .RETURN(oa_ErrorCode_kOK);
 
       WHEN("the manager's identifier is queried") {
-        const openassetio::Str actualIdentifier = cManagerInterface.identifier();
+        const openassetio::Identifier actualIdentifier = cManagerInterface.identifier();
```

Comment:
Doh! I did check that PR to see if I could spot ones I missed, but missed them there as well!

The `ManagerTest.cpp` change I'm not so sure about, since the string there is really used as binary storage rather than an identifier.

Reply:
Updated in 2fd3f122